.env.example

```diff
@@ -1,7 +1,47 @@
-DEBUG=
+# Unused atm (jest sets this to `test`)
 NODE_ENV=development
-TMPDIR=$TMPDIR
-#To allow testing different executables in the bin tests
-#PK_TEST_DOCKER_IMAGE=$image #Specify the docker image that the `docker-run.sh` uses
-#PK_TEST_COMMAND=scripts/docker-run.sh #Specify the executable we want to test against
-#PK_TEST_COMMAND_DOCKER=DOCKER #Specify if the test is for docker for filtering relevant tests
+
+# Debug node modules - https://nodejs.org/api/cli.html#node_debugmodule
+# NODE_DEBUG=
+
+# Debug node native modules - https://nodejs.org/api/cli.html#node_debug_nativemodule
+# NODE_DEBUG_NATIVE=
+
+# Path to PK executable to override tests/bin target
+# PK_TEST_COMMAND=
+
+# If set, indicates that `PK_TEST_COMMAND` is targetting docker
+# PK_TEST_COMMAND_DOCKER=
+# Login credentials for GitLab's container registry
+# Automatically injected from CI/CD
+CI_REGISTRY_IMAGE=registry.gitlab.com/matrixai/open-source/js-polykey
+CI_REGISTRY_USER=
+CI_REGISTRY_PASSWORD=
+
+# If set, indicates that `PK_TEST_COMMAND` is targetting docker
+# PK_TEST_COMMAND_DOCKER=
+# Accessing AWS for testnet.polykey.io and mainnet.polykey.io deployment
+AWS_DEFAULT_REGION='ap-southeast-2'
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+# Container registry domain
+PK_AWS_ECS_REGISTRY='015248367786.dkr.ecr.ap-southeast-2.amazonaws.com'
+# Container repository
+PK_AWS_ECS_IMAGE="$PK_AWS_ECS_REGISTRY/polykey"
+# ECS cluster name
+PK_AWS_ECS_CLUSTER='polykey-testnet'
+# ECS service name
+PK_AWS_ECS_SERVICE='polykey-testnet'
```

Comment:
These have to be set in the `.gitlab-ci.yml` where necessary when interacting with ECR and ECS. Note that `scripts/deploy-image.sh` uses `CI_REGISTRY_IMAGE`, thus one must set `CI_REGISTRY_IMAGE="$PK_AWS_ECS_IMAGE` when one wants to deploy to ECR, and not gitlab's container registry.